### PR TITLE
Agnes feature

### PR DESCRIPTION
**Pull request by asrobang1**

### CODE REVIEW

File: agnes_file.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+first line
+second line
+surprise! third line
```
Comment (jordan-mb) [inline agnes_file.txt:3]: Too scary

Comment (jordan-mb) [reply agnes_file.txt:3]: Blah blah

Comment (asrobang1) [reply agnes_file.txt:3]: Sunday scaries

Comment (jordan-mb) [reply agnes_file.txt:3]: ldjfldksjfl

**Review by jordan-mb — APPROVED** [review]
Good!

**Pull request merged**